ui/components/src/EChart/EChart.tsx

```diff
@@ -127,6 +127,9 @@ export const EChart = React.memo(function EChart<T>({
     if (containerRef.current === null || chartElement.current !== null) return;
     chartElement.current = init(containerRef.current, theme, { renderer: renderer ?? 'canvas' });
     if (chartElement.current === undefined) return;
+    // https://echarts.apache.org/en/api.html#echarts.connect
+    chartElement.current.group = 'dashboard';
```

Comment:
It might be a good idea to add a new prop, `syncGroup` that allows you to this programmatically.

```tsx
// These are in sync
<EChart {...props} syncGroup="panel-group-1" />
<EChart {...props} syncGroup="panel-group-1" />

// then these are in sync
<EChart {...props} syncGroup="panel-group-2" />
<EChart {...props} syncGroup="panel-group-2" />
```

Reply:
yeah for sure, I definitely was planning to add an easy way to opt out, I like the syncGroup idea though to let the parent have control over the group name too